src/main/java/ch/njol/skript/aliases/AliasesProvider.java

```diff
@@ -171,7 +173,7 @@ public AliasesProvider(int expectedCount, @Nullable AliasesProvider parent) {
 		this.aliases = new HashMap<>(expectedCount);
 		this.variations = new HashMap<>(expectedCount / 20);
 		this.aliasesMap = new AliasesMap();
-		this.materials = new ArrayList<>();
+		this.materials = new ObjectOpenHashSet<>();
```

Comment:
What is the default size here? I wonder if we would benefit from using a higher initial capacity value? :)

Reply:
The default size is 16 — I left it out as I wasn't sure how many materials we'd end up with. In any case, I did not notice any meaningful performance impact from having to resize the table.